### DISCUSSION TITLE
PCBs in Germayny/EU

### DISCUSSION THREAD

**Discussion by McBlade** [nebhead/PiFire#64] [General]
Hello,

first of all, great job! I like it very much. I am in the moment playing with the hardware a little bit. For this I have ordered V3 PCB designed from weberbox. With this order I had to buy 5 pcs... I just need one. So if there is someone in Germany or Europe who needs it, just write a message. I just want to have the money I had paid. I have got all 5 pcs for 10 Euro. So I have four for 2 Euro each, additional shipping.

I hope this is ok, to do it here. But I want to get the PCBs in use of this wonderfull project.

Stefan

**Comment by nebhead** [comment]
Welcome to the project!  Perfectly fine to order and share boards.  I have had others ask about boards in Europe (and Germany specifically) so it would great if you can offer some to anyone interested locally.  I'm sure @weberbox would be pleased to know that his boards are being used all the way across the world.  :)

**Comment by thomasstrach** [comment]
Hi all,

I am based in Germany and have the project up and running here.
So far I still have 3 PCBs left here from my order. Please contact me on my email if you are interested.
Best regards 
Thomas
E-Mail: ***@***.***
